### DISCUSSION TITLE
How to exclude a language coming from an engine

### DISCUSSION THREAD

**Discussion by drallieiv** [ember-intl/ember-intl#1573] [Q&A]
I have an app than includes an ember engine with 3 language (en, fr, es)
But in the final app I only want 2 languages (en, fr)

When I build and I look at what is inside the app js (not the vendor, the other one)
I still have all the keys and translations for es.

I've tried adding exclude at several places, but I can't find how to make it work.
How can I remove a language from the final build ?

I've searched around, and the key seems to be in broccoli trees, but I am having hard time trying to find out how to remove something as it seems additive by nature.

technically, there is 2 levels : 
* app > engine A > shared engine with translation
* app > engine B > shared engine with translation

### Environment

ember-cli: 3.8.3
node: 10.9.0

**Comment by sandstrom** [comment]
I'm doing some issue gardening 🌱🌿 🌷 and came upon this issue. Maybe we could move this over to the [discussions](https://github.com/ember-intl/ember-intl/discussions).

By closing some old issues we reduce the list of open issues to a more manageable set.
